Projects/hCoreUI/Sources/hForm/hFloatingTextField.swift

```diff
@@ -253,7 +256,7 @@ public struct hFloatingTextField<Value: hTextFieldFocusStateCompliant>: View {
 
 class TextFieldVM: ObservableObject {
     @Published var textField: UITextField?
-    let disposeBag = DisposeBag()
+    var cancellables = Set<AnyCancellable>()
```

Comment:
I think it would be better to use 
`var buttonCancellable: AnyCancellable?
`
instead  of
`var cancellables = Set<AnyCancellable>()
`

Reply:
Doesn't work